lib/ts/recipe/session/recipe.ts

```diff
@@ -40,7 +40,7 @@ export default class Session extends RecipeModule<unknown, unknown, unknown, any
                     config.onHandleEvent(event);
                 }
 
-                this.notifyListeners(event);
+                void this.notifyListeners(event);
```

Comment:
Why are we adding `void` in here and other places?

Reply:
`void` signifies that we are intentionally ignoring the return value of `notifyListeners`, which in this case is a Promise that we do not want to await/handle. This way we can still have these "floating promises" and not trigger the lint rule.